lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestSimpleEventHandlers.cs

```diff
@@ -48,7 +48,7 @@ public void TestSetup()
         [Timeout(180 * 1000)]
         public void RunSimpleEventHandlerOnLocalRuntime()
         {
-            string testFolder = DefaultRuntimeFolder + TestNumber++;
+            string testFolder = DefaultRuntimeFolder + Guid.NewGuid().ToString("N").Substring(4);
```

Comment:
Just don't want too long folder name. We also clean up after the test. The opportunity of duplication is already very low.